content/body/switch.php

```diff
@@ -35,8 +35,8 @@
 <p>This code is based on information from the
     <a href="https://developer.mozilla.org/en-US/">MDN</a> article on
     <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_switch_role">Using
-      the switch role</a>.  The switch reports the checked state as "on" or "off" in VoiceOver
-    and "checked" or "unchecked" in NVDA and ChromeVox.  In order to make some consistency among user agents, an <code>aria-describedby</code> on the switch can state the "on/off" state to all screen readers. This description is also given visually, to make it obvious what the state is for sighted users.  Developers could hide this text with the <a href="screen-reader-only-text.php"><code>sr-only</code> class</a>, and put "off" and "on" labels on sides of the right and left sides of the component if they wish instead. 
+      the switch/toggle role</a>.  The switch/toggle reports the checked state as "on" or "off" in VoiceOver
```

Comment:
There is no such thing as an ARIA toggle role, which is why it just said "switch" in this sentence.

```suggestion
      the switchrole</a>.  The switch/toggle reports the checked state as "on" or "off" in VoiceOver
```